zellij-utils/src/input/unit/layout_test.rs

```diff
@@ -45,7 +45,10 @@ fn default_layout_merged_correctly() {
                 borderless: true,
                 parts: vec![],
                 split_size: Some(SplitSize::Fixed(1)),
-                run: Some(Run::Plugin(Some("tab-bar".into()))),
+                run: Some(Run::Plugin(Some(RunPlugin {
+                    path: "tab-bar".into(),
+                    _allow_exec_host_cmd: false,
```

Comment:
If you want, you could `#[derive(Default)]` on `RunPlugin` and do:

```rs
RunPlugin {
    path: "tab-bar".into(),
    ..Default::default()
}
```
I get the feeling that the `RunPlugin` struct will be growing and this should prevent us from needing to update this file every time that happens.

Reply:
Really good advice, thanks.